apps/opik-frontend/src/components/pages/ExperimentsShared/AddExperimentDialog.tsx

```diff
@@ -145,6 +143,11 @@ const AddExperimentDialog: React.FunctionComponent<
   AddExperimentDialogProps
 > = ({ open, setOpen }) => {
   const workspaceName = useAppStore((state) => state.activeWorkspaceName);
+  const { data: user } = useUser();
```

Comment:
I don't think we can use this `user` not in `plugins`. Maybe, we could discuss it with @andriidudar on Monday.  But somehow, we should put it into plugins. Maybe, create a new one or using already existing one